### PR TITLE
produce cards with javascript dynamically

### PR DESCRIPTION
**Pull request by piesieomane**
This pull request consists of injecting directly to the DOM of the work section using DOM manipulation with javascript and also coding the popup modal.

My partner said no need for him to commit to this, and I have reached out to the success team

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -92,151 +96,11 @@ <h2 class="name piesie">I'm Piesie Glad to see you!</h2>
...
-            <li class="css">css</li>
-            <li class="javascript">javascript</li>
-          </ul>
-          <button>See Project</button>
-        </div>
-      </section>
-    </section>
+    <div class="bg-popup">
+      <div class="popup"></div>
+    </div>
```
Comment (keroloussamy) [inline index.html:103]: - [ ] Kindly on the mobile version, I can't see the close button to close the window, Kindly fix it to match the original design.

image

**Review by keroloussamy — CHANGES_REQUESTED** [review]
Hi @piesieomane, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To highlight 🎯

- No linter errors.✔️
- Good code design and implementation. ✔️

## Required Changes ♻️
_Check the comments under the review._




Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (piesieomane) [reply index.html:103]: please kindly check the browser you are using

Comment (piesieomane) [reply index.html:103]: Okay, lemme do that please

File: style.css [hunk 2]
```diff
@@ -151,10 +189,37 @@ section .icon-right {
   cursor: pointer;
 }
 
+.pic-container .cards-pic {
+  width: 100%;
+  height: 85%;
+}
+
```
Comment (vechicin) [inline style.css:196]: [OPTIONAL] I would recommend you fix your popup image width. This is the way it's currently looking on my end.

Screen Shot 2022-06-15 at 8 15 46

**Review by vechicin — APPROVED** [review]
# Hi @piesieomane ✋, 

Your project is complete! 🎯 🧠 💯 
There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights ⚡ 

- [x] Good job implementing the changes required by the previous reviewer! 

## Optional suggestions 📂 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 2 above)
Comment (piesieomane) [reply style.css:196]: yes, the image was difficult to style. it stretches. If you know any procedure to help me style it, I will be grateful @vechicin

**Pull request merged**